srt-protocol/src/connection/mod.rs

```diff
@@ -303,6 +306,7 @@ impl DuplexConnection {
         self.timers.reset_exp(now);
 
         self.stats.rx_all_packets += 1;
+        self.stats.rx_all_bytes += size as u64;
```

Comment:
This is not correct--rx_all_bytes is supposed to include header sizes--I kinda prefer my approach to these metrics that I take in #134, but if you have a preference I'm willing to discuss.

Reply:
Ok, this was the bytes read in from receive buffer, I'm fine with pulling this out if you'd prefer.